code/__DEFINES/power.dm

```diff
@@ -11,9 +11,12 @@
 #define SOLAR_TRACK_AUTO 2
 
 ///conversion ratio from joules to watts
-#define WATTS / 0.002
+//#define WATTS / 0.002
 ///conversion ratio from watts to joules
-#define JOULES * 0.002
+//#define JOULES * 0.002
+
+///The amount of energy, in joules, a standard powercell has.
+#define STANDARD_CELL_CHARGE 1e6
```

Comment:
1e6 is confusing. You should do 1000 JOULES. Where JOULES is a defi e can be typed as 1e3 so people who aren't good at match can understand better

Reply:
Why would JOULES be equal to 1,000? 1 = 1 joule.

Reply:
I realised I forgot to remove commented out code. You can ignore that.

Reply:
Even though 1 joule is = 1 watt then create a define like

`# define JOULE * 1` and you can go all the way to giga joule.

Trush me, a little bit of semantics can improve readibility a long way

Reply:
The define's comment specifies it's in joules.

Reply:
Comments are insufficient and error prone to changes.

We have defines like DESICOND whose value is also 1. Same is needed for joules

Reply:
I'll define the JOULE, JOULES, WATT and WATTS simply to state they're the default energy and power units. Seems like people assumed they are considering at least 3 of the conservation of energy issues were related to people assuming powercells used joules.